test-automation/api-tests/test.py

```diff
@@ -46,83 +59,181 @@ def load_keys(cli) -> list:
         for file in key_content:
             if file.endswith(".key"):
                 key_file_path = f"{os.getcwd()}/{args.keys_dir}/{key}/{file}"
-                new_account_name = f"api_test_{i}"
-                cli.remove_address(new_account_name)
+                new_account_name = f"_test_key_{random_num}_{i}"
+                CLI.remove_address(new_account_name)
```

Comment:
let's make the CLI remove keys, later make this consume that hmy keys --remove (something like that)